### PR TITLE
PIL-1276_added avalanche chain in etherspot service

### PR DESCRIPTION
**Pull request by vignesha22**
Added Avalanche Chain config and integrated Avalanche chain in etherspot service by initialising EtherspotSDK for avalanche chain. Also added balance fetch for that chain and stored as an etherspot wallet

### CODE REVIEW

**Review by Yash-Chaubal — APPROVED** [review]
Lgtm!

File: src/services/etherspot.js [hunk 1]
```diff
@@ -600,6 +618,16 @@ export class EtherspotService {
...
+        .getTransactions({ account: sdk.state.accountAddress })
+        .then(({ items }) => items)
+        .catch((error) => {
+          reportErrorLog('getTransactionsByAddress -> getTransactions failed', { address, chain, error });
+          return null;
+        });
+    }
+
     return sdk
       .getTransactions({ account: address })
```
Comment (IAmKio) [inline src/services/etherspot.js:632]: Do you think it's better to change this line to `.getTransactions({ account: sdk.state.accountAddress })` as opposed to creating a `if` statement just for the one chain above on line 621?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just one minor change, other than that - LGTM!

File: src/services/etherspot.js (hunk 1 above)
Comment (vignesha22) [reply src/services/etherspot.js:632]: yeah it is

File: src/services/etherspot.js [hunk 2]
```diff
@@ -121,9 +122,15 @@ export class EtherspotService {
...
-      const env = networkName !== NetworkNames.Kovan ? EnvNames.MainNets : EnvNames.TestNets;
-      this.instances[networkName] = new EtherspotSdk(privateKey, { env, networkName, projectKey: PROJECT_KEY });
-
+      const env =
+         networkName !== NetworkNames.Kovan && networkName !== NetworkNames.Matic && networkName !== NetworkNames.Fuji
+           ? EnvNames.MainNets
+           : EnvNames.TestNets;
+      this.instances[networkName] = new EtherspotSdk(privateKey, {
+        env,
+        networkName: networkName === NetworkNames.Matic ? NetworkNames.Mumbai : networkName,
```
Comment (IAmKio) [inline src/services/etherspot.js:131]: Did you hardcode Mumbai testnet here for Polygon?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Please see my latest comment

**Review by IAmKio — APPROVED** [review]
LGTM 🙏🏾

**Pull request merged**